RecoTracker/FinalTrackSelectors/plugins/MultiTrackSelector.cc

```diff
@@ -561,10 +575,13 @@ void MultiTrackSelector::processMVA(edm::Event& evt, const edm::EventSetup& es,
     return;
   }
 
+  if(!useMVA_[selIndex] && !writeIt)return;
```

Comment:
this looks odd though (at least without trying to understand the rest of the code)